Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM python:3.8-slim AS conpot-builder
+
+# Install dependencies
+#RUN apt update \
+#    && apt install ipmitool git python3-dev build-base py-cffi libxslt-dev libffi-dev py3-pip \
+#    && apt install mysqlclient-dev
+
+RUN apt-get update && apt-get install -y \
+    libmariadb-dev \
+    gcc \
+    && rm -rf /var/lib/apt/lists/*
+
+# Copy the app from the host folder (probably a cloned repo) to the container
+RUN adduser --disabled-password --gecos "" conpot
+
+RUN mkdir /opt/conpot
+COPY . /opt/conpot/
+RUN chown conpot:conpot -R /opt/conpot
+
+# Install Python requirements
+USER conpot
+
+WORKDIR /opt/conpot
+
+ENV PATH=$PATH:/home/conpot/.local/bin
+
+RUN pip3 install --user --no-cache-dir -r requirements.txt
+
+# Install the Conpot application
+RUN python3 setup.py install --user --prefix=
```

Comment:
```suggestion
# Install Conpot
USER conpot
ENV PATH=$PATH:/home/conpot/.local/bin
RUN pip3 install --user --no-cache-dir /opt/conpot
```

This should be equivalent but slightly more robust against changes to `setup.py` or `requirements.txt`.